src/widgets/training-program/ui/training-program.module.scss

```diff
@@ -0,0 +1,91 @@
+.training-program {
+  &.column-2 {
+    display: flex;
+    flex-direction: row;
+    gap: 100px;
+    align-items: center;
+    justify-content: flex-start;
+
+    .left {
+      display: flex;
+      flex: 1 1 55%;
+      flex-direction: column;
+      gap: 26px;
+
+      max-width: 55%;
+
+      & span {
+        font-weight: $font-weight-bold;
+      }
+
+      div {
+        padding-bottom: 18px;
+
+        p {
+          padding-bottom: 0;
+          font-weight: $font-weight-bold;
+        }
+      }
+
+      & > .button {
```

Comment:
Shouldn't we have removed the use of & in styles?

Reply:
please, see https://github.com/rolling-scopes/site/pull/609#discussion_r1822105087

Reply:
Thank you for clarifying. I understood the recommendation the same way Misha did.